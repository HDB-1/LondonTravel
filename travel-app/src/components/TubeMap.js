import React from 'react';
import tubemap from '../assets/tube_map.png';
import '../styling/TubeMap.css';

export default function TubeMap() {
    return (
        <div>
            <img src={tubemap} alt="Tube Map"/>
        </div>
    )
}
