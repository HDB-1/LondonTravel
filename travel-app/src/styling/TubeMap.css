img {
    width: 100vh;
    margin: 10px;
}
