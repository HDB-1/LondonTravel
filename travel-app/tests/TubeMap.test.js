import React from 'react';
import { shallow } from 'enzyme';
import TubeMap from '../src/components/TubeMap';

describe("shallow TubeMap", () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<TubeMap />)
    })
    it("should render correctly", () => {
        expect(wrapper).toMatchSnapshot();
    })
})