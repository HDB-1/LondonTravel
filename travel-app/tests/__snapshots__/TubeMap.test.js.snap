// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow TubeMap should render correctly 1`] = `
<div>
  <img
    alt="Tube Map"
    src="test-file-stub"
  />
</div>
`;
